modules/ossm-installing-multi-primary-multi-network-topology.adoc

```diff
@@ -0,0 +1,292 @@
+----
+
+. Deploy the sample applications on the West cluster:
+
+.. Create a sample application namespace on the West cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get ns sample --context "${CTX_CLUSTER2}" || oc create --context="${CTX_CLUSTER2}" namespace sample
+----
+
+.. Label the application namespace to support sidecar injection by running the following command:
++
+[source,terminal]
+----
+$ oc label --context="${CTX_CLUSTER2}" namespace sample istio-injection=enabled
+----
+
+.. Deploy the `helloworld` application:
+
+... Create the `helloworld` service by running the following command:
++
+[source,terminal]
+----
+$ oc apply --context="${CTX_CLUSTER2}" \
+    -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/helloworld/helloworld.yaml \
+    -l service=helloworld -n sample
+----
+
+... Create the `helloworld-v2` deployment by running the following command:
++
+[source,terminal]
+----
+oc apply --context="${CTX_CLUSTER2}" \
+  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/helloworld/helloworld.yaml \
+  -l version=v2 -n sample
+----
+
+.. Deploy the `sleep` application by running the following command:
++
+[source,terminal]
+----
+$ oc apply --context="${CTX_CLUSTER2}" -n sample \
+     -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/sleep/sleep.yaml
+----
+
+. Wait for the sample applications on the West cluster to return the "Ready" status condition by running the following command:
++
+[source,terminal]
+----
+$ oc --context="${CTX_CLUSTER2}" wait --for condition=available -n sample \
+     deploy/helloworld-v2 deploy/sleep
+----
+
+.Verification
+
+. For the East cluster, send 10 requests to the `helloworld` service by running the following command:
++
+[source,terminal]
+----
+$ for i in {0..9}; do
+    oc exec --context="${CTX_CLUSTER1}" -n sample deploy/sleep -c sleep \
+       -- curl -sS helloworld.sample:5000/hello;
+  done
+----
++
+Verify that you see responses from both clusters. This means you will see version 1 and version 2 of the service in the responses. 
+
+. For the West cluster, send 10 requests to the `helloworld` service:
++
+[source,terminal]
+----
+$ for i in {0..9}; do
+    oc exec --context="${CTX_CLUSTER2}" -n sample deploy/sleep -c sleep \
+       -- curl -sS helloworld.sample:5000/hello;
+  done
+----
++
+Verify that you see responses from both clusters. This means you will see version 1 and version 2 of the service in the responses. 
+
+.Cleanup
+
+. Optional: Remove the clusters from a development environment by running the following commands:
++
+[source,terminal]
+----
+oc delete istios default --context="${CTX_CLUSTER1}"
+oc delete ns istio-system --context="${CTX_CLUSTER1}" 
+oc delete ns sample --context="${CTX_CLUSTER1}"
+oc delete istios default --context="${CTX_CLUSTER2}"
+oc delete ns istio-system --context="${CTX_CLUSTER2}" 
+oc delete ns sample --context="${CTX_CLUSTER2}"
```

Comment:
Add `$` and combine the commands so that you don't have to split this into 6 code snippets:
```suggestion
$ oc --context="${CTX_CLUSTER1}" delete istio/default ns/istio-system ns/sample && \
  oc --context="${CTX_CLUSTER2}" delete istio/default ns/istio-system ns/sample
```

Reply:
done